ABOUT.md

```diff
@@ -1,9 +1,72 @@
+* *many languages* (Python, Octave, R, Java/Scala, Lua, C#, Ruby, etc) and *platforms* (Linux/Unix, MacOS and Windows), easy integration with other ML libraries
 
-Shogun got initiated in 1999 by [Soeren Sonnenburg](http://sonnenburgs.de/soeren) and [Gunnar Raetsch](http://www.raetschlab.org/) (that's where the name ShoGun originates from). It is now developed by a much larger Team cf. [website](http://shogun-toolbox.org/page/about/ourteam) and [AUTHORS](http://www.github.com/shogun-toolbox/shogun/wiki/AUTHORS), and would not have been possible without the patches and bug reports by various people. Statistics on Shogun's development activity can be found on [ohloh](https://www.openhub.net/p/shogun).
+#### Shogun is state-of-the-art
+
+* *efficient and competitive* implementation (from standard to cutting edge algorithms), modern software architecture in C++
+* easy combination of *multiple data representations, algorithm classes and general purpose tools* for rapid prototyping of data pipelines
+
+### Shogun is open source
+
+* *free* and *community-based* [ML training and skills](#useshoguntolearn) not just for students
+
+You can try Shogun in our [cloud](https://cloud.shogun.ml/hub/home) (no installation required) or find out more about our [mission](#mission), [features](#features) and [history](#shogunhistory).
+
+
+
+
+
+
+## <a name="mission"></a> Mission
+
+Machine learning is powerful. Within the last two decades, machine learning has emerged from an eccentric academic field to transforming the way we live our everyday life. Its methods influence society and its tools have a billion-dollar impact on global economy. We want to make sure that the underlying machine learning algorithms are transparent and accessible. We believe that ML should not be a privilege of corporations or political power with the money to buy technical knowledge. 
+Anyone - scientists, data enthusiasts, non-profit organisations, hackers, citizens, journalists, students - should be able to use ML tools for free.
+
+The Shogun ML toolbox promotes this mission through its features (*accessible*, *open source* and *good*) and a focus on ML education and developing its contributors.
+
+## <a name="features"></a> Shogun features - *accessible*, *open source* and *good*
+
+### Accessible
+
+ * Shogun is **not limited to a single language**: you can use the toolbox through a unified interface from C++, Python, Octave, R, Java, Lua, C#, Ruby, etc. See [examples](http://shogun.ml/cookbook/latest/). Thus we are independent of trends in computing languages and importantly, it allows you to use Shogun as a vehicle to expose your algorithm to multiple communities. We use [SWIG](http://www.swig.org/) to enable *bidirectional* communication between C++ and all target languages.
```

Comment:
Grammar error
"Thus WE are independentm ...., IT allows" (what does IT reference?)